### PR TITLE
Added Breakout! Game

### PR DESCRIPTION
**Pull request by bitaashna**
## Related Issue
  - #616 

Closes: #616 

### Describe the changes you've made
**Breakout! Game**
Game where you control a paddle with the arrow keys to bounce a ball up to break bricks. This app uses the HTML5 canvas element and API

**Project Specifications**

- Drawn elements on canvas
- Used canvas paths to draw shapes
- Added animation
- Move paddle on arrow key press 
- Added collision detection
- Keep score

### Describe if there is any unusual behaviour of your code(Write `NA` if there isn't)
NA

### Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] New and existing unit tests pass locally with my changes.

### Screenshots

breakout

### CODE REVIEW

**Comment by bitaashna** [comment]
@urvashi-code1255 @Vishal-raj-1 @harshgupta20  Please review it!

**Review by urvashi-code1255 — COMMENTED** [review]
@bitaashna Add your project details in main script.js

**Comment by bitaashna** [comment]
@urvashi-code1255 Already done!
Screenshot from 2021-04-02 18-34-49

File: assets/js/Breakout.js [hunk 1]
```diff
@@ -0,0 +1,289 @@
...
+							}
+						} else {
+							if (x <= b.x + brickWidth + ballRadius && x - dx > b.x + brickWidth + ballRadius) {
+								dx = -dx;
+								b.status = 0;
+								score++;
+								if (score == brickRowCount * brickColumnCount) {
+									alert("You win!");
+									document.location.reload();
+								}
```
Comment (urvashi-code1255) [inline assets/js/Breakout.js:219]: @bitaashna Remove all these unwanted empty lines

Comment (bitaashna) [reply assets/js/Breakout.js:176]: Done!

**Comment by bitaashna** [comment]
@urvashi-code1255 I have made the changes.Please review it!

**Review by urvashi-code1255 — APPROVED** [review]
Its looking good to me @bitaashna :)

**Pull request merged**